controllers/drpolicy_controller.go

```diff
@@ -231,35 +249,44 @@ func (r *DRPolicyReconciler) createOrUpdateManifestWorkForVRC(ctx context.Contex
 		})
 
 		if err != nil {
-			klog.Error(err, "failed to create/update ManifestWork: %s", manifestWorkName)
+			logger.Error("Failed to create/update ManifestWork", "ManifestWorkName", manifestWorkName, "error", err)
 			return err
 		}
 
-		klog.Infof("ManifestWork created for %s", vrcName)
+		logger.Info("ManifestWork created/updated successfully", "ManifestWorkName", manifestWorkName, "VolumeReplicationClassName", vrcName)
 	}
 
 	return nil
 }
 
 func (r *DRPolicyReconciler) fetchClusterFSIDs(ctx context.Context, peer *multiclusterv1alpha1.MirrorPeer, clusterFSIDs map[string]string) error {
+	logger := r.Logger.With("method", "fetchClusterFSIDs", "MirrorPeer", peer.Name)
```

Comment:
```suggestion
	logger := r.Logger.With("MirrorPeer", peer.Name,"ClusterName", pr.ClusterName)
```

Reply:
Not sure if you wanted the logger to be added inside the loop but i've added it inside the loop since it will be at most 2 operations.